### DISCUSSION TITLE
Support for Laravel 9

### DISCUSSION THREAD

**Discussion by philippgerard** [codecyclernl/notion-php-sdk#14] [Ideas]
Probably no changes needed, but currently requires <9

``` codecycler/notion 0.10.0 requires illuminate/support ^6.0|^7.0|^8.0 -> found illuminate/support[v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require```

**Comment by SebastiaanKloos** [comment]
@philippgerard I just released a new version. Should be fixed in https://github.com/codecyclernl/notion-php-sdk/releases/tag/0.11.0
